src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-cluster-form/smb-cluster-form.component.html

```diff
@@ -202,75 +202,144 @@
         class="cds--btn__icon"></svg>
       </button>
     </div>
-
+</ng-container>
     <!-- Placement -->
     <ng-container *ngIf="orchStatus.available">
       <div class="form-item">
         <cds-select
-          label="Placement"
-          for="placement"
-          formControlName="placement"
-          id="placement"
+          formControlName="auth_mode"
+          label="Authentication Mode"
+          i18n-label
+          cdRequiredField="Authentication Mode"
+          id="auth_mode"
+          [invalid]="smbForm.controls.auth_mode.invalid && smbForm.controls.auth_mode.dirty"
+          [invalidText]="authModeError"
+          (change)="onAuthModeChange()"
+          [disabled]="isEdit"
+          helperText="Active-directory authentication for domain member servers and User authentication for
+          Stand-alone servers configuration."
```

Comment:
Suggest "Active Directory authentication for domain member servers and User authentication for standalone servers"